cfgov/paying_for_college/jinja2/paying-for-college/disclosure.html

```diff
@@ -83,7 +83,7 @@ <h2 class="verify_prompt">
                         </h2>
                         <form class="verify_form">
                             <div class="verify_school">
-                                {{school.primary_alias}}
+                              {% if school%}{{school.primary_alias}}{% endif %}
```

Comment:
Instead of all the conditionals, would it make sense to have something like this at the top:

```
{% set school_primary_alias = school.primary.alias if school else '' %}
```

And then `{{ school_primary_alias }}` in the spots where it is used?

If you do keep the conditions, perhaps format them the same as `{% if school %}`, instead of a mix of that and `{% if school%}`

Reply:
Sure, I just did it with a mass replace in vim ^^;

Reply:
Done in 79c3f